### ISSUE TITLE
Podium byline modification

### ISSUE THREAD

**Issue by chrismplus** [mplusmuseum/mplusmuseum-stories#229]
Hi @rdunk — we have an upcoming Podium article with numerous 'respondents'. The 'By' in the byline won't be applicable in this instance. Pauline would prefer something like "Respondents: Person A, Person B and Person C". 

The article in question is the unpublished piece in Issue 1. 

- **Request 1:** can we account for this in the CMS? An option to list "Respondents:" in the byline _instead_ of "By".
- **Request 2 (nice to have but not a must):** numerous respondents will reply to the prompt at the top of the piece. Could we include a non-hyperlinked prompt such as: "more to come" that only appears on the Article-level, i.e. it would be suppressed at the Issue and Podium landing pages? I understand there's a bit of logic to account for in this second request. And therefore we don't see it as completely necessary. Just wanted to see whether you saw an easy fix for this...

**Comment by rdunk** [comment]
Request 1 is now implemented and deployed to the staging CMS and Heroku staging app for you to test.

You can now choose a byline type for articles, so they will use either the 'byline' (default) or 'bylineRespondents' translations in the journal translations file. Hope that makes sense!

Re: Request 2, I'm sure it will be possible (pretty much everything is) but not sure I follow the request exactly, what do you mean by prompt? Do you have any visuals you could share that might give me a clearer picture of what you mean?

**Comment by chrismplus** [comment]
Hi @rdunk, I've tested this in some detail on the staging. I've uncovered the following two issues for Request 1 (let's leave aside request 2 for now): 

1. If you look at _Museums, Managers of Consciousness_ on the Staging heroku site (Podium page) you'll notice that the listings for Respondents works. However, if you look at On Smallness in Hong Kong Art at the top of the page you'll see that Winnie Wong is now listed as a 'respondent' instead of the normal "By..." byline. I think this is because I listed her as a test as a 'respondent'. While it's unlikely for now that we'll have a double-up with a past contributor, I just want to make sure we can fix this bug in case. 

eg:
![image](https://user-images.githubusercontent.com/26731347/51722760-72360380-2091-11e9-88ff-b712ac1129f8.png)

2. I added the Chinese translation for respondents (= 回應者：) into the language file in ProcessWire. This seems to have broken the respondent listing for the Chinese-language pages, however:

Chinese: 
![image](https://user-images.githubusercontent.com/26731347/51722801-9eea1b00-2091-11e9-95d2-1d5d42911a4e.png)

English:
![image](https://user-images.githubusercontent.com/26731347/51722811-a6a9bf80-2091-11e9-87f9-1a2c451ae47f.png)

Let me know whether you need any extra details? As noted, let's leave aside the second request for now. I think we're covering all bases through listing the respondents via a new byline.

**Comment by rdunk** [comment]
Hey @chrismplus, thanks for testing this. To respond to the the two issues you're encountering:

1. I think I had set the _On Smallness in Hong Kong Art_ article to use the "Respondents" byline as a test when I deployed the change, I've switched it back now. I don't think setting the byline for one author should affect any other articles. Am I understanding this issue correctly?

2. Ah, so for this you still need to add the shortcodes in the dynamic text to render properly: `{DATE}` and/or `{AUTHORS}` as necessary. I've just changed it to `= 回應者：{AUTHORS}` for now (not sure this is the correct place for the shortcode though!) and it seems to render the list out OK.

Sure, let me know about the second request if anything changes. Happy to discuss.
